pkg/operator/handler.go

```diff
@@ -23,6 +23,12 @@ type Handler struct {
 	ManifestDir string
 }
 
+func (h *Handler) SetConfig(conf *v1.NetworkConfig) {
+	h.syncLock.Lock()
+	h.config = conf
+	h.syncLock.Unlock()
```

Comment:
preferred style would be

    h.syncLock.Lock()
    defer h.syncLock.Unlock()
    h.config = conf

? Sure, it doesn't matter yet, but it might given future changes to the function